### PR TITLE
Target Android 11

### PR DESCRIPTION
**Pull request by ShivamPokhriyal**
Product Note: Added support for Android 11.

### CODE REVIEW

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462205/)

File: app/src/org/commcare/activities/components/FormEntryInstanceState.java [hunk 1]
```diff
@@ -106,8 +103,7 @@ public void loadFromIntent(Intent intent) {
         if (intent.hasExtra(KEY_FORM_RECORD_DESTINATION)) {
             this.mFormRecordDestination = intent.getStringExtra(KEY_FORM_RECORD_DESTINATION);
         } else {
-            Logger.log(LogTypes.SOFT_ASSERT, "Access outside scoped storage in loading current form.");
-            mFormRecordDestination = ODKStorage.FORM_RECORD_PATH;
+            throw new RuntimeException("Access outside scoped storage in loading current form.");
```
Comment (shubham1g5) [inline app/src/org/commcare/activities/components/FormEntryInstanceState.java:106]: the wordings of this doesn't make sense with the code to access outside scope storage deleted. Can we just say more on the lines of "Invalid Intent definition, no extra with name 'KEY_FORM_RECORD_DESTINATION` present in the intent"

File: app/src/org/commcare/tasks/FormLoaderTask.java [hunk 2]
```diff
@@ -235,24 +235,7 @@ private void setupFormMedia(String formMediaPath, File formXmlFile) {
...
-            String formFileName = formXmlFile.getName().substring(0, formXmlFile.getName().lastIndexOf("."));
-
-            // Set jr://... to point to /sdcard/odk/forms/filename-media/
-            ReferenceManager.instance().addSessionRootTranslator(
-                    new RootTranslator("jr://images/", "jr://file/forms/" + formFileName + "-media/"));
-            ReferenceManager.instance().addSessionRootTranslator(
-                    new RootTranslator("jr://audio/", "jr://file/forms/" + formFileName + "-media/"));
-            ReferenceManager.instance().addSessionRootTranslator(
-                    new RootTranslator("jr://video/", "jr://file/forms/" + formFileName + "-media/"));
+            throw new RuntimeException("Access outside scoped storage in form media setup.");
```
Comment (shubham1g5) [inline app/src/org/commcare/tasks/FormLoaderTask.java:238]: Same comment as above on wordings of exception.

**Review by shubham1g5 — DISMISSED** [review]

**Comment by dimagimon** [comment]
[32% (0.0%) vs master 32%](https://jenkins.dimagi.com/job/commcare-android-pr-job-builder/462212/)

**Review by shubham1g5 — APPROVED** [review]

**Pull request merged**